resources/ts/Pages/SpecDocSheet/Edit.tsx

```
@@ -16,307 +21,284 @@ import { Flash } from "@/types/Flash";
-                    href={route("specDocSheets.preview", {
-                        projectId: specDoc.projectId,
-                        specDocId: specDoc.id,
-                        specDocSheetId: specDocSheet.id,
-                    })}
-                    target="_blank"
-                    className="spec-doc-sheet-edit__preview-link"
-                >
-                    Preview
-                    <img src="/img/icon-new-window.png" />
-                </a>
+      return () => clearTimeout(timer);
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)